packages/svg-serializer/test.js

```diff
@@ -2,30 +2,72 @@ const test = require('ava')
 const {CAG, CSG} = require('@jscad/csg')
 const serializer = require('./index.js')
 
-test('serialize ag/csg objects to svg (path: simple)', function (t) {
+test('serialize CAG objects to SVG paths', function (t) {
+  const cag1 = new CAG()
```

Comment:
as a general note, I think it is better to give more explicit names to variables: ie
- instead of **cag1** : *emptyCag*
- instead of **cag2** : **rectangle**,  **testRectangle**  etc
- instead of **cag3** : **offsetRectangle**,  **translatedRectangle**
- etc
it helps keep the tests more understandable and human friendly :)

Reply:
True. However, previous tests were using expected or data. :)